src/OpenIddict.EF/OpenIddictExtensions.cs

```diff
@@ -0,0 +1,40 @@
+﻿/*
+ * Licensed under the Apache License, Version 2.0 (http://www.apache.org/licenses/LICENSE-2.0)
+ * See https://github.com/openiddict/core for more information concerning
+ * the license and the contributors participating to this project.
+ */
+
+using System;
+using Microsoft.AspNet.Identity;
+using Microsoft.AspNet.Identity.EntityFramework;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Internal;
+using OpenIddict;
+using OpenIddict.Models;
+
+namespace Microsoft.AspNet.Builder {
+    public static class OpenIddictExtensions {
+        public static OpenIddictBuilder AddEntityFrameworkStores<TUser, TApplication, TRole, TKey>([NotNull] this OpenIddictBuilder builder)
```

Comment:
We only use 1 store, rename `AddEntityFrameworkStores` to `AddEntityFrameworkStore`.